app/src/main/java/org/p2p/wallet/receive/tokenselect/ReceiveTokensPresenter.kt

```diff
@@ -51,33 +51,33 @@ class ReceiveTokensPresenter(
                     Constants.ETH_SYMBOL
                 )
             )
-            solToken = tokensForReceiveBanner[0]
-            ethToken = tokensForReceiveBanner[1]
             view.setBannerTokens(
-                firstTokenUrl = solToken?.iconUrl.orEmpty(),
-                secondTokenUrl = ethToken?.iconUrl.orEmpty()
+                firstTokenUrl = ERC20Tokens.SOL.tokenIconUrl.orEmpty(),
+                secondTokenUrl = ERC20Tokens.ETH.tokenIconUrl.orEmpty()
             )
             observeTokens()
             observeMappedTokens()
         }
     }
 
     private fun preparePinedWormholeTokens(): List<TokenData> {
-        return ERC20Tokens.values().mapNotNull { erc20Token ->
-            interactor.findTokenDataByAddress(erc20Token.mintAddress)?.let { token ->
-                TokenData(
-                    mintAddress = token.mintAddress,
-                    name = erc20Token.replaceTokenName ?: token.tokenName,
-                    symbol = erc20Token.replaceTokenSymbol ?: token.tokenSymbol,
-                    iconUrl = erc20Token.tokenIconUrl ?: token.iconUrl,
-                    decimals = token.decimals,
-                    coingeckoId = erc20Token.coingeckoId,
-                    isWrapped = false,
-                    serumV3Usdc = null,
-                    serumV3Usdt = null
-                )
+        return ERC20Tokens.values().toMutableList()
+            .apply { removeAll(listOf(ERC20Tokens.LUNA, ERC20Tokens.UST)) }
```

Comment:
maybe filter will work better here?

Reply:
can do with filter

just thought that removing it would be better than checking every item with filter